docstring_parser/google.py

```diff
@@ -38,6 +38,8 @@ class Section(namedtuple("SectionBase", "title key type")):
 
 
 GOOGLE_TYPED_ARG_REGEX = re.compile(r"\s*(.+?)\s*\(\s*(.*[^\s]+)\s*\)")
+MULTIPLE_PATTERN = re.compile(r'(\s*[^:\s]+:)|(.*\]:.*)')
```

Comment:
I'd replace `.*` in the other half with `[^:]*`

Reply:
If we use `[^:]*` it wouldn't capture past the second colon in the following.

```
Optional[Mapping[str, List[int]]]: A description: with a colon
```

It doesn't change the behaviour as we're not capturing anything.

Would you like me to make the change?

Reply:
I meant `r'(\s*[^:\s]+:)|([^:]*\]:.*)')` to ensure there are no colons in the type hint part

Reply:
Gotcha